### ISSUE TITLE
Accuracy and Consistency about Introspect.

### ISSUE THREAD

**Issue by LemonQC** [sergiotasconmorales/imp_vqa#3]
I follow the steps and run the code, obtaining the following results. I found the Consistency is lower than report.
![image](https://github.com/sergiotasconmorales/imp_vqa/assets/30914380/81e9520c-98db-4773-8459-c920a3966646)
![image](https://github.com/sergiotasconmorales/imp_vqa/assets/30914380/30ef7f49-c72e-4ec4-89ed-1b8ff07a8ed2)

**Comment by sergiotasconmorales** [comment]
@LemonQC Yes, like I mention in the section "Computing Consistency," this is likely due to the steps involved in the preparation of the Introspect data (training BERT on SNLI then fine-tuning). The annotated relations end up being a bit different because the initialization is not the same every time. If you try with the data I obtained the first time, you will get very similar results to the ones in the paper (keep in mind the results in the paper are the average of several models, so some variance is expected). The links to download the data from Zenodo are at the beginning of the section "Data."
I'm still working on ensuring that the data preparation process results in the same data every time.

**Comment by LemonQC** [comment]
Thank you for your timely response. This is a great job, and I will continue to identify issues. I also hope you can help me.

**Comment by sergiotasconmorales** [comment]
Thank you. The issue you found made me check the prepared data again, and it looks like the relations are not being predicted correctly by BERT. So I will focus on fixing that. 
If I can help you I will :)
